native/membrane_video_compositor/src/elixir_bridge/elixir_structs.rs

```diff
@@ -466,7 +337,7 @@ mod tests {
                     Object::Layout(Layout {
                         inputs,
                         resolution: LayoutOutputResolution::Name(a),
-                        implementation: 42,
+                        implementation: (42, 42),
```

Comment:
This is a little inconsistent. In the `Object::Layout` reference to a specific implementation of layouts is called `implementation`, while in the `Object::Texture` list of the same type, references (references to plugin-provided implementations of Layout/Transformation) are called `transformations`.

Maybe we should normalize that.

Reply:
Renamed it to `params`. with this change at least the name is accurate.